src/content/en/2021/resource-hints.md

```diff
@@ -439,101 +439,141 @@ Prioritizing third-party resources over your own content is potentially a warnin
+    </tr>
+    <tr>
+      <td>
+        www.google-analytics.com
+      </td>
+      <td class="numeric">1.2%</td>
+      <td class="numeric">3.9%</td>
+      <td class="numeric">0.2%</td>
+      <td class="numeric">5.3%</td>
+    </tr>
+    <tr>
+      <td>
+          www.googletagmanager.com
+      </td>
+      <td class="numeric">1.9%</td>
+      <td class="numeric">2.7%</td>
+      <td class="numeric">0.2%</td>
+      <td class="numeric">4.8%</td>
+    </tr>
+    <tr>
+      <td>static.parastorage.com</td>
+      <td class="numeric">-</td>
+      <td class="numeric">-</td>
+      <td class="numeric">4.7%</td>
+      <td class="numeric">4.7%</td>
+    </tr>
+    <tr>
+      <td>ajax.googleapis.com</td>
+      <td class="numeric">2.2%</td>
+      <td class="numeric">1.6%</td>
+      <td class="numeric">0.3%</td>
+      <td class="numeric">4.1%</td>
+    </tr>
+    <tr>
+      <td>www.google.com</td>
+      <td class="numeric">2.7%</td>
+      <td class="numeric">1.0%</td>
+      <td class="numeric">0.1%</td>
+      <td class="numeric">3.8%</td>
+    </tr>
+    <tr>
+      <td>images.squarespace-cdn.com</td>
+      <td class="numeric">-</td>
+      <td class="numeric">3.5%</td>
+      <td class="numeric">-</td>
+      <td class="numeric">3.5%</td>
+    </tr>
+    <tr>
+      <td>cdnjs.cloudflare.com</td>
+      <td class="numeric">1.6%</td>
+      <td class="numeric">1.0%</td>
+      <td class="numeric">0.4%</td>
+      <td class="numeric">2.9%</td>
+    </tr>
+    <tr>
+      <td>monorail-edge.shopifysvc.com</td>
+      <td class="numeric">2.0%</td>
+      <td class="numeric">0.8%</td>
+      <td class="numeric">-</td>
+      <td class="numeric">2.8%</td>
+    </tr>
+    <tr>
+      <td>fonts.shopifycdn.com</td>
+      <td class="numeric">-</td>
+      <td class="numeric">1.1%</td>
+      <td class="numeric">1.0%</td>
+      <td class="numeric">2.1%</td>
+    </tr>
+  </tbody>
+</table>
+
 
-<figcaption>{{ figure_link(caption="Most popular third-party connections using resource hints.", sheets_gid="1254656281", sql_file="preload_host_by_url.sql") }}</figcaption>
+<figcaption>{{ figure_link(caption="Most popular third-party connections using resource hints on mobile devices.", sheets_gid="1254656281", sql_file="preload_host_by_url.sql") }}</figcaption>
 </figure>
 
-Analyzing the table above, 42.7% of all pages which include a `preload` hint are preloading resources hosted on adservice.google.com, while s.w.org (WordPress) is the most popular domain for `dns-prefetch` and `fonts.gstatic.com` for the `preconnect` directive.
+Analyzing the table above, 36.7% of all pages which include a `preload` hint are preloading resources hosted on adservice.google.com. s.w.org (WordPress) is the most popular domain for `dns-prefetch` and is added by default on new WordPress.org installations. Google Fonts related services on `fonts.gstatic.com` and `fonts.googleapis.com` are the two most popular hosts for the `preconnect` directive.
```

Comment:
Trying to get confirmation on this statement.

> is added by default on new WordPress.org installations

Reply:
https://core.trac.wordpress.org/ticket/34292

> By default, wp_resource_hints() prints hints for "s.w.org" (the WordPress.org CDN) and for all scripts and styles which are enqueued from external hosts.

Reply:
The `s.w.org` domain is the CDN for serving emoji SVG images (provided by Twemoji) when WordPress detects (via its inlined `emoji-loader.js`) that the browser doesn't support native emoji characters. This CDN has been used since WordPress 4.6. 

```suggestion
Analyzing the table above, 36.7% of all pages which include a `preload` hint are preloading resources hosted on adservice.google.com. The s.w.org host is the most popular domain for `dns-prefetch` and is used on WordPress sites (since version 4.6) for the loading of SVG images from its Twemoji CDN, when the browser is detected to not support native emoji characters. Google Fonts related services on `fonts.gstatic.com` and `fonts.googleapis.com` are the two most popular hosts for the `preconnect` directive.
```